Fix/prepare release (#435)
retest/recheck-web: README.md

### recheck-web seems to be slow when executing. What can I do about that?

> recheck-web itself is not slow. But screenshot creation is. Lucky enough, we only use the screenshot for documentation, not for the actual comparison. It will be only create a screenshot of the viewport and you can disable the screenshot creation completely. For more information, please refer to the [documentation](https://docs.retest.de/recheck-web/usage/configuration/).

Reviewer: ```suggestion
recheck-web itself is not slow. But screenshot creation is. Lucky enough, we only use the screenshot for documentation, not for the actual comparison. Per default, we only create a screenshot of the given viewport and you can also disable the screenshot creation completely. For more information, please refer to our [documentation](https://docs.retest.de/recheck-web/usage/configuration/).
```
